packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/angular/reqauthspecific.md

```diff
@@ -1,16 +1,9 @@
 The `OktaAuthGuard` can protect a route to a single component as well with `canActivate` property.
 
-
-1. Import the `ProfileComponent` component into `app-routing.module.ts`:
-
-   ```ts
-   import { ProfileComponent } from './profile/profile.component';
-   ```
-
-2. Add the following to the `routes` array:
+1. Add the `canActivate` property to the `routes` array in `app-routing.module.ts`:
 
    ```ts
    { path: 'profile', component: ProfileComponent, canActivate: [OktaAuthGuard] }
    ```
 
-This single `/profile` route is now protected.
+The single `/profile` route is protected directly, and when navigating to the page it prompts you to sign in first before routing you to the `ProfileComponent`.
```

Comment:
```suggestion
The single `/profile` route is protected directly. When you navigate to the page, you must sign in before being routed to the `ProfileComponent`.
```

Reply:
Hey @isaacdurham-okta! How would you recommend wording this if the user is already authenticated? In that case, the user isn't prompted to sign in before navigating the route.

Reply:
I think we're trying to highlight that you can access this route if authenticated and, if not, you're prompted to sign in. How about:

_The single `/profile` route is protected directly. If you're not authenticated and navigate to this page, you must sign in before being routed to the `ProfileComponent`._ 

Is that too clunky? @alisaduncan @isaacdurham-okta

Reply:
LGTM!

Reply:
LGTM. Sorry, I missed your comments yesterday!